ftw/permissionmanager/configure.zcml

```diff
@@ -58,12 +58,5 @@
     <!-- Patches, using monkeypatcher -->
     <!-- Let ftw.solr's reindexObjectSecurity be loaded first -->
     <include zcml:condition="installed ftw.solr" package="ftw.solr" />
```

Comment:
Since we do not have a monkey patch anymore in this package, we can clean up:
- remove this line: the purpose was to have a deterministic patch order - but we are not patching anymore here, so no need to order something
- remove the comments above (58/59)
- remove monkey namespace declaration
- remove `collective.monkeypatcher` from `setup.py`
- remove other traces if there are any